nixos/modules/services/x11/window-managers/i3.nix

```diff
@@ -23,6 +23,23 @@ in
           If left at the default value, $HOME/.i3/config will be used.
         '';
       };
+
+      debug = mkOption {
+        default = false;
+        example = true;
+        type = types.bool;
+        description = "Enable debug/verbose logging (see -V option)";
+      };
+
+      logFile = mkOption {
+        default = null;
+        example = "$HOME/.i3/i3log";
+        type = types.string;
```

Comment:
The previous behaviour was to put the logs in display-manager.unit logs, that is, writing everything to stdout within the xsession script. This is what I wanted to preserve with the default null value.
Tested then thought smart to add the type option... Apologies for the annoyance.